docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,308 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
-    corresponding API references for information about the `qiskit.primitives` and `qiskit_aer.primitives` options.
-</Admonition>
+## Overview
 
-## V2 changes
-Options are specified differently in the V2 primitives in these ways:
+<span id="options-structure"></span>
+### Structure
 
-- `SamplerV2` and `EstimatorV2` now have separate options classes. You can see the available options and update option values during or after primitive initialization.
-- Instead of the `set_options()` method, V2 primitive options have the `update()` method that applies changes to the `options` attribute.
-- If you do not specify a value for an option, it is given a special value of `Unset` and the server defaults are used.
-- For V2 primitives, the `options` attribute is the `dataclass` Python type.  You can use the built-in `asdict` method to convert it to a dictionary.
+When calling the primitives, you can pass in options by using an options class or a dictionary. Commonly used options, such as `resilience_level`, are at the first level. Other options are grouped into different categories, such as `execution`. See the [Set primitive options](specify-runtime-options) topic for full details.
 
-## Instantiate the Options class (V1)
+<span id="options-defaults"></span>
+### Defaults
 
-In the example below, we create an instance of the `Options` class. `optimization_level` is a first-level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
+For V2 primitives, if you do not specify a value for an option, it is given a special value of `Unset` and the server default value is used. Thus, the default value will be the same regardless of your code version.
+For V1 primitives, if you do not specify a value for an option, the Qiskit Runtime client's default value is used. In V1, the default value is dependant on the code version.
 
-```python
-from qiskit_ibm_runtime import Options
+The tables in the [Options classes summary](#options-classes) section lists the default values.
 
-options = Options(optimization_level=1, environment={"log_level": "INFO"})
-```
+<span id="options-precedence"></span>
+## Set options
 
-The `Options` class supports auto-complete. Once you create an instance of the `Options` class, you can use auto-complete to see what options are available. If you choose one of the categories, you can use auto-complete again to see what options are available under that category.
+Options can be defined before a primitive is constructed and passed to the primitive, which will make a copy of them. As shown below, this can be done either as a nested dictionary, or by using the options classes. However, this is not necessary, because after the primitive is constructed, the options that it owns can be changed. You can choose the workflow that works best for your application.
```

Comment:
> However, this is not necessary,

It's unclear to me what is unnecessary? Passing options to a primitive prior to constructing it?